templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -173,9 +173,19 @@
 						{{.locale.Tr "repo.issues.new.open_projects"}}
 					</div>
 					{{range .OpenProjects}}
-						<a class="item muted sidebar-item-link" data-id="{{.ID}}" data-href="{{.Link}}">
+						{{$ProjectID := .ID}}
+						{{$checked := false}}
+						{{range $.Issue.Projects}}
+							{{if eq .ID $ProjectID}}
+								{{$checked = true}}
+							{{end}}
+						{{end}}
+					<a class="item muted sidebar-item-link{{if $checked}} checked{{end}}" href="#" data-id="{{.ID}}" data-href="{{$.RepoLink}}/projects/{{.ID}}">
```

Comment:
Does this link work for org-level projects?

Reply:
Modified